t/client-connect.t

```diff
@@ -0,0 +1,64 @@
+use strict;
+use warnings;
+use OPCUA::Open62541 ':all';
+use POSIX qw(sigaction SIGALRM);
+
+use Net::EmptyPort qw(empty_port);
+use Test::More tests => 12;
+use Test::NoWarnings;
+use Test::Warn;
+
+my $s = OPCUA::Open62541::Server->new();
+ok($s, "server");
+
+my $sc = $s->getConfig();
+ok($s, "config server");
+
+my $port = empty_port();
+my $r = $sc->setMinimal($port, "");
+is($r, STATUSCODE_GOOD, "minimal server config");
+
+my $c = OPCUA::Open62541::Client->new();
+ok($c, "client");
+
+my $cc = $c->getConfig();
+ok($c, "config client");
+
+$r = $cc->setDefault();
+is($r, STATUSCODE_GOOD, "default client config");
+
+
+my $pid = fork // die "Unable to fork: $!\n";
+
+if ( !$pid ) {
+    my $running = 1;
+    sub handler {
+	$running = 0;
+    }
+
+    # Perl signal handler only works between perl statements.
+    # Use the real signal handler to interrupt the OPC UA server.
+    # This is not signal safe, best effort is good enough for a test.
+    my $sigact = POSIX::SigAction->new(\&handler);
+    sigaction(SIGALRM, $sigact);
+    alarm(1);
+
+    # run server and stop after one second
+    $s->run($running);
+
+    exit 0;
```

Comment:
When exiting from a forked child, use POSIX::_exit.  exit may execute cleanup code twice, once in the parent and one in the child.  This has very strange effects with END blocks.